src/Mover.php

```diff
@@ -77,16 +78,39 @@ public function movePackage(Package $package)
                         $contents = $replacer->replace($contents);
 
                         if ( is_a($replacer, ClassmapReplacer::class)) {
-                            $replacedClasses = array_merge($replacedClasses, $replacer->replacedClasses);
+                            $this->replacedClasses = array_merge($this->replacedClasses, $replacer->replacedClasses);
                         }
 
                         $filesystem->put($targetFile, $contents);
                     }
                 }
             }
         }
+    }
+
+    public function replaceClassmapNames()
+    {
+        $classmap_path = $this->workingDir . $this->config->classmap_directory;
+        $finder = new Finder();
+        $finder->files()->in($classmap_path);
 
-        // Replace all replaced class names throughout the classmap dependencies here
+        $filesystem = new Filesystem(new Local($this->workingDir));
 
+        foreach ($finder as $file) {
+            $file_path = str_replace($this->workingDir, '', $file->getRealPath());
+            $contents = $filesystem->read($file_path);
+
+            foreach( $this->replacedClasses as $original => $replacement ) {
+                $contents = preg_replace_callback(
+                    '/\W('.$original.')(?:\(|\:\:)/U',
```

Comment:
Thanks @schlessera, I would love to make this as bullet proof as I can, but don't want this to distract too much from actually implementing the classmap handler. Improving the replacing logic can obviously be an ongoing process. Can you please post this as an issue, so I can handle it accordingly ( - it would be amazing if the GitHub UI allowed me to promote this comment to an issue, directly from here)?